### PR TITLE
v1.1: Added clarity on what we mean by service

### PR DESCRIPTION
**Pull request by mike-finopsorg**
Discussed in  #414 we need to clarify what FOCUS means by `service`

### CODE REVIEW

**Review by ahullah — APPROVED** [review]

File: specification/overview.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ The following principles were considered while building the specification.
 * Where possible, use consistent names that will naturally create associations between related columns in the specification.
 * Column naming must strictly follow the [column naming conventions](#columnnamingconvention).
 * Use established standards (e.g., ISO8601 for dates, ISO4217 for currency).
+* The use of the term `service` used throughout this specification will be used to reference offerings by cloud service providers/SaaS Vendors/Internal Platforms. [Managed Service Providers (MSPs)](#glossary:managed-service-provider) will be used when refering to MSPs or Resellers.
```
Comment (flanakin) [inline specification/overview.md:75]: Fwiw, my only concern was about `ServiceName` and `ServiceCategory`. Both of those will apply to MSPs as well if they have extra charges, right? I think the only inconsistency with "FinOps tools and services" is that FOCUS uses "service" to represent _both_ "tools" and "services".

**Comment by jpradocueva** [comment]
The group agreed to move it to v1.1

**Review by ljadvey — APPROVED** [review]

**Comment by shawnalpay** [comment]
Discussed in Oct 31 Members call.  Will follow up with @mike-finopsorg and @flanakin to determine whether we carry this forward or not.

**Comment by jpradocueva** [comment]
### Comment from Members' call on Oct 31:
**Analysis**: This issue, dating back to April, involves defining "service" for consistency in documentation, but it remains unresolved and somewhat outdated.
**Agreements:** Shawn will follow up with Mike and  Michael to decide if this issue should be updated or closed.

**Pull request closed without merging**